R-packages/covidcast/R/covidcast.R

```diff
@@ -48,6 +48,9 @@ COVIDCAST_BASE_URL <- 'https://api.covidcast.cmu.edu/epidata/'
 # Max rows returned by API
 MAX_RESULTS <- 1000000
 
+# Cache the httr covidcast_meta response when available
+META_RESPONSE <<- NA
```

Comment:
```suggestion
META_RESPONSE <- NA
```
This one can stay `<-`; it's only inside the function that we need to save to the parent scope